frontend/assets/stylesheets/components/_payment-paypal.scss

```diff
@@ -221,6 +221,7 @@
         background-color: #fff;
         top: 2px;
         margin-left: -12px;
+        border: 1px solid $c-neutral4;
```

Comment:
Ok so this is technically the same as `guss-colour(neutral-4)`, but tbh I have no idea what our convention is at this point, we have a mix throughout the codebase. Do you prefer to use the guss functions, or the defined colours as you've done here?